src/migration.js

```diff
@@ -54,24 +54,24 @@ module.exports = class Migration {
       })();
     }
 
-    return require(this.path);
+    return import(this.path);
   }
 
   /**
    * Executes method `up` of migration.
    *
-   * @returns {*|Promise}
+   * @returns {Promise}
    */
-  up() {
+  async up() {
     return this._exec(this.options.upName, [].slice.apply(arguments));
   }
 
   /**
    * Executes method `down` of migration.
    *
-   * @returns {*|Promise}
+   * @returns {Promise}
    */
-  down() {
+  async down() {
     return this._exec(this.options.downName, [].slice.apply(arguments));
```

Comment:
There is no reason to make the "up" and "down" methods async if you aren't going to await anything.

_exec still returns a promise, so you could either remove async from the signature, or you could await "this._exec(...)".